easybuild/easyconfigs/p/python-parasail/python-parasail-1.1.16-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = "PythonPackage"
+
+name = 'python-parasail'
+version = '1.1.16'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pypi.org/project/parasail/'
+description = """ This package contains Python bindings for parasail. """
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://pypi.python.org/packages/source/p/parasail']
+sources = [{'filename': SOURCE_TAR_GZ, 'download_filename': 'parasail-%(version)s.tar.gz'}]
+checksums = ['00d2be6b78368667376a7a0eae265a1a9c35ed210a546902340f621f32f28078']
+
+builddependencies = [
+    ('parasail', '2.4'),
+    ('wheel', '0.31.1', versionsuffix),
+]
+dependencies = [
+    ('Python', '3.6.6'),
+]
+
+download_dep_fail = True
+use_pip = True
+
+# make sure setup.py finds the parasail library
+preinstallopts = "ln -s $EBROOTPARASAIL/lib/libparasail.so parasail/libparasail.%s && " % SHLIB_EXT
```

Comment:
I see this was already like this in the existing easyconfig by @smoors, but shouldn't `SHLIB_EXT` be used for both occurrences of the shared library extension `.so`?

Reply:
yes, this would probably better written as
```
preinstallopts = "ln -s $EBROOTPARASAIL/lib/libparasail.%s parasail/libparasail.%s && " % (SHLIB_EXT, SHLIB_EXT)
```